Dockerfile

```diff
@@ -61,7 +63,7 @@ RUN curl -fsSL https://packages.microsoft.com/config/debian/10/packages-microsof
   && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s --  --profile minimal -y \
   && cargo --version \
   && cargo install cargo-hack \
-  && gem install cocoapods --no-document \
+  && gem install -g -no-document\
```

Comment:
this command is incorrect (see the original command) and the space should be preserved